doc/getting_started/index.rst

```diff
@@ -179,292 +142,276 @@ We'll also install Zephyr's multi-purpose west tool.
-   you try building another sample.
+      .. code-block:: bash
 
-.. rst-class:: numbered-step
+         cd ~/zephyrproject/zephyr
+         west build -p auto -b <your-board-name> samples/basic/blinky
 
-Flash and Run the Application
-*****************************
+   .. group-tab:: macOS
 
-#. Connect a USB cable between the board and your development computer.
-   (Refer to the specific :ref:`boards` documentation if you're not sure
-   which connector to use on the board.)
-#. If there's a switch, turn the board on.
-#. Flash the blinky application you just built using the command:
+      .. code-block:: bash
 
-   .. tabs::
+         cd ~/zephyrproject/zephyr
+         west build -p auto -b <your-board-name> samples/basic/blinky
 
-      .. group-tab:: Ubuntu
+   .. group-tab:: Windows
 
-         .. code-block:: bash
+      .. code-block:: bat
 
-            west flash
+         cd %HOMEPATH%\zephyrproject\zephyr
+         west build -p auto -b <your-board-name> samples\basic\blinky
 
-         If the flash command fails, and you've checked your
-         board is powered on and connected to the right on-board USB connector,
-         verify you've granted needed access permission by
-         :ref:`setting-udev-rules`.
+The ``-p auto`` option automatically cleans byproducts from a previous build
+if necessary, which is useful if you try building another sample.
 
-      .. group-tab:: macOS
+.. rst-class:: numbered-step
 
-         .. code-block:: bash
+Flash the Sample
+****************
 
-            west flash
+Connect your board, usually via USB, and turn it on if there's a power switch.
+If in doubt about what to do, check your board's page in :ref:`boards`.
 
-      .. group-tab:: Windows
+Then flash the sample using :ref:`west flash <west-flashing>`:
 
-         .. code-block:: bat
+.. code-block:: console
 
-            west flash
+   west flash
 
+You may need to install additional :ref:`host tools <debug-host-tools>`
+required by your board. The ``west flash`` command will print an error if any
+required dependencies are missing.
 
-The application will start running and you'll see blinky in action. The
-actual blinking LED location is board specific.
+If you're using blinky, the LED will start to blink as shown in this figure:
 
 .. figure:: img/ReelBoard-Blinky.gif
    :width: 400px
    :name: reelboard-blinky
 
-   Phytec reel_board running blinky
-
+   Phytec :ref:`reel_board <reel_board>` running blinky
 
 Next Steps
 **********
 
-Now that you've got the blinky sample running, here are some next steps
-for exploring Zephyr:
+Here are some next steps for exploring Zephyr:
 
-* Try some other :ref:`samples-and-demos` that demonstrate Zephyr
-  capabilities.
-* Learn about :ref:`application` and more details about :ref:`west`.
-* Check out :ref:`beyond-GSG` for information about advanced setup
-  alternatives and issues.
+* Try other :ref:`samples-and-demos`.
```

Comment:
I suggest also linking to the debugging section here, which is likely one of the things users want to try first.